File: samples_generated_code/samples_2/phi4/baseline_code_java/code_row_104.java
public class ArgvIndex {
    public static void main(String[] args) {
        if (args.length < 2) {
            System.out.println("Usage: java ArgvIndex <index>");
            return;
        }

        try {
            int index = Integer.parseInt(args[0]);
            if (index < 1 || index >= args.length) {
                System.out.println("Error: Index out of range. Valid indices are from 1 to " + (args.length - 1));
            } else {
                System.out.println(args[index]);
            }
        } catch (NumberFormatException e) {
            System.out.println("Error: The first argument must be a valid integer.");
        }
    }
}

File: samples_generated_code/samples_2/phi4/baseline_code_py/code_row_104.py
import sys

def main():
    if len(sys.argv) < 2:
        print("Error: No index provided.")
        return
    
    try:
        index = int(sys.argv[1]) - 1
    except ValueError:
        print("Error: Index must be an integer.")
        return
    
    if len(sys.argv) <= index + 2:
        print(f"Error: Argument at index {index + 1} does not exist.")
        return

    print(sys.argv[index + 2])

if __name__ == "__main__":
    main()